db/migrate/20150107115419_create_questions.rb

```diff
@@ -0,0 +1,14 @@
+class CreateQuestions < ActiveRecord::Migration
+  def change
+    unless ActiveRecord::Base.connection.table_exists? 'questions'
+      create_table :questions do |t|
+        t.integer :user_id
```

Comment:
t.integer :user_id could be replaced by t.references :user and foreign_key will be automatically added into record